src/Request/Request.php

```diff
@@ -393,6 +397,22 @@ private function userAgent()
         );
     }
 
+    /**
+     * Gives the terminus command.
+     *
+     * @return string
+     */
+    private function terminusCommand()
+    {
+        $argv = $GLOBALS['argv'];
+        $command = $argv[1];
+        if (in_array($command, self::SENSITIVE_COMMANDS)) {
+            return $command;
+        } else {
+            return implode(" ", array_slice($argv, 1));
```

Comment:
I don't like this. It seems like `SENSITIVE_COMMANDS` is not reliable enough to ensure that we're not leaking something that we don't want to. A prime example of this is the Terminus Secrets Manager plugin. I don't think this can be a blocklist.

I think it's probably find to just return the command name, and omit all of the arguments from the trace header. The command parameters that are relevant to the operation are already included in some request or another for the command. If you extract data on the server side from the other request parameters, you have the advantage that they have already been put into named parameters, so you know what is a site uuid, and etc..  The allowlist implementation might be more verbose than the blocklist implementation, but it has the advantage that you're not going to accidentally log something that you didn't mean to (e.g. a customer's personal access token or ssh private key).